packages/styleguide/src/styles.scss

```diff
@@ -1,3 +1,6 @@
+@import "~@a-ui/acpaas/dist/main.min.css";
+@import "~@a-ui/flexboxgrid/dist/flexboxgrid.min.css";
```

Comment:
There were problems with the cdn not loading the css at the time I wrote this code, but this approach obviously works better for dev too, since you can now link the libs

Reply:
Can you please revert this change? I know CDNs have their disadvantages, but the problems you are talking about have been resolved and we try to keep loading times as lean and mean as possible.